specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+# Charge Subcategory
+
+Charge Subcategory is a detailed descriptor within the billing and usage reports that acts as a supplementary detail to the main ChargeType column. It essentially dissects the primary category of charge to indicate more specific information about how each cost item relates to predefined pricing commitments.
```

Comment:
Does "detailed descriptor" make it sound like it'd be a textual description? Suggesting some tweaks to simplify and align with ChargeType...
```suggestion
Charge Subcategory indicates what kind of usage or adjustment the row represents. Charge Subcategory is a supplemental detail to the Charge Type. It provide additional context to describe the primary category of charge.
```